packages/core/src/components/Sheet/styles.ts

```diff
@@ -147,7 +147,7 @@ export const styleSheetSheet: StyleSheet = ({ color, pattern, ui, unit }) => ({
   },
 
   headerShadow: {
-    boxShadow: ui.boxShadow,
+    borderBottom: ui.borderThick,
```

Comment:
why thick?

Reply:
It's the same height to what the shadow was. IMO, I like it.